frontend-react/src/components/header/ReportStreamNavbar.tsx

```diff
@@ -82,25 +92,63 @@ export const ReportStreamNavbar = ({
     };
     const defaultMenuItems = [
         <div className="primary-nav-link-container">
-            <a className="primary-nav-link" href="/product" key="product">
-                Product
+            <a
+                className={primaryLinkClasses(!!useMatch("/product/*"))}
```

Comment:
`useMatch` returns a path object if it finds something and `null` if nothing. we're typecasting this to `true` `false`.

Reply:
with `useMatch` we can add the `*` at the end of the query so that it boldens the link for any pages that begin with the proper route.